package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "npm run lint:css && npm run lint:js",
     "lint:css": "stylelint './src/client/**/*.js'",
     "lint:js": "eslint --ignore-path .gitignore ./",
-    "server:build": "babel ./src/server --out-dir ./build && cp ./src/server/index.pug ./build/index.pug",
+    "server:build": "babel ./src/common --out-dir ./build && cp ./src/server/index.pug ./build/index.pug",
```

Comment:
This compiles only files from the `common` folder now (maybe you didn't notice as the ones from `server` where still in the `build` folder). You can check the `babel` documentation but it should be possible to add multiple paths.

Reply:
oh, no I didn't mean to leave this in. Will remove